apps/remote_control/lib/lexical/remote_control/code_mod/ast.ex

```diff
@@ -14,6 +18,171 @@ defmodule Lexical.RemoteControl.CodeMod.Ast do
   end
 
   defp parse(s) when is_binary(s) do
-    ElixirSense.string_to_quoted(s, 1, 6, token_metadata: true)
+    case Sourceror.string_to_quoted(s, token_metadata: true, columns: true) do
+      {:ok, code, _comments} ->
+        {:ok, code}
+
+      error ->
+        error
+    end
+  end
+
+  def zipper_at(%Document{} = document, %Document.Position{} = position) do
```

Comment:
Shouldn't this function be private?

Reply:
No, I think this could be really useful for code mods.